web/src/components/Button/Button.scss

```diff
@@ -53,6 +53,10 @@
     background-color: #00afa4;
     color: #ffffff;
   }
+  &.black {
+    background-color: black;
+    color: #ffffff;
+  }
```

Comment:
what's up with this one?

Reply:
I did that when I was making mucking around with the Report Issue button in the bottom corner but it is not necessary.